pkgs/development/python-modules/htmlmin/default.nix

```diff
@@ -2,28 +2,36 @@
   lib,
   buildPythonPackage,
   fetchPypi,
-  pythonAtLeast,
+  setuptools,
+  standard-cgi,
 }:
 
 buildPythonPackage rec {
   pname = "htmlmin";
   version = "0.1.12";
-  format = "setuptools";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "50c1ef4630374a5d723900096a961cff426dff46b48f34d194a81bbe14eca178";
+    hash = "sha256-UMHvRjA3Sl1yOQAJapYc/0Jt/0a0jzTRlKgbvhTsoXg=";
   };
 
-  # Tests run fine in a normal source checkout, but not when being built by nix.
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    standard-cgi
+  ];
+
+  # pypi tarball does not contain tests
```

Comment:
Maybe we should ask upstream for tags.

Reply:
Last update was 8 years ago, I don't have high hopes.